thirdeye/thirdeye-frontend/app/pods/rootcause/template.hbs

```diff
@@ -112,18 +124,20 @@
             {{/if}}
 
             {{#if (eq activeTab "metrics")}}
-              <h3>Metrics</h3>
+              {{#if isLoadingAggregates}}
```

Comment:
one loader instead of individual loaders? 
One problem I saw with this implementation is that the container could potentially be very big, and the loader might not be in view of the user.

Reply:
multiple spinners indicate loading even if the tabs are up-to-date already. We'll probably need individual loading status. then again, this view will turn into a table anyways